style.css

```diff
@@ -1,8 +1,408 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  border: none;
+  outline: none;
+  scroll-behavior: smooth;
+  font-family: "lato", sans-serif;
+}
+
+html {
+  line-height: 1.15;
+  -webkit-text-size-adjust: 100%;
+  font-size: 62.5%;
+  overflow-x: hidden;
+}
+
```

Comment:
I noticed that while your sections are looking good, there seems to be a slight alignment issue, Please fix it

![alignment](https://github.com/ibtisam-adil/Capstone/assets/118625824/260d56fa-e944-4d7d-abb2-e61ebfde3b37)

Reply:
fixed